protected/humhub/modules/admin/messages/ru/user.php

```diff
@@ -18,84 +18,95 @@
-  'Disable' => 'Выключить',
+  'Delete pending registrations?' => 'Удалить ожидающие регистрации?',
+  'Delete spaces which are owned by this user' => 'Удалить все сообщества, принадлежащие этому пользователю',
+  'Delete' => 'Удалить',
+  'Deleted invitation' => 'Удалённое приглашение',
+  'Deleted users' => 'Удалённые пользователи',
+  'Deleted' => 'Удалено',
+  'Disable' => 'Отключить',
+  'Disabled users' => 'Отключённые пользователи',
   'Disabled' => 'Отключён',
-  'Disabled users' => 'Отключенные пользователи',
-  'Do not change placeholders like {displayName} if you want them to be automatically filled by the system. To reset the email content fields with the system default, leave them empty.' => 'Не меняйте условное название (placeholders), такие как {displayName}, если вы хотите, чтобы они автоматически заполнялись системой. Чтобы сбросить поля содержимого электронной почты до системных значений по умолчанию, оставьте их пустыми.',
-  'Edit' => 'Изменить',
-  'Edit category' => 'Изменить категорию',
+  'Do not change placeholders like {displayName} if you want them to be automatically filled by the system. To reset the email content fields with the system default, leave them empty.' => 'Пожалуйста, не меняйте заполнители типа {DisplayName}, если вы хотите, чтобы они автоматически заполнялись системой. Чтобы вернуть значения по умолчанию, оставьте поля пустыми.',
+  'Do you really want to delete pending registrations?' => 'Вы действительно хотите удалить ожидающие регистрации?',
+  'Do you really want to re-send the invitations to pending registration users?' => 'Вы действительно хотите повторно отправить приглашения пользователям, ожидающим регистрации?',
+  'Edit category' => 'Редактировать категорию',
   'Edit profile category' => 'Изменить категорию профиля',
   'Edit profile field' => 'Изменить поле профиля',
   'Edit user: {name}' => 'Редактировать пользователя: {name}',
+  'Edit' => 'Изменить',
   'Email all selected' => 'Отправить по электронной почте все выбранные',
+  'Enable individual profile permissions' => 'Включить индивидуальные права и разрешения профиля',
   'Enable' => 'Включить',
-  'Enable individual profile permissions' => 'Включить разрешения для отдельных профилей',
   'Enabled' => 'Включён',
   'First name' => 'Имя',
   'Group Manager' => 'Менеджер группы',
   'Group not found!' => 'Группа не найдена!',
   'Group user not found!' => 'Участник группы не найден!',
   'Groups' => 'Группы',
-  'Hello {displayName},
+    'Hello {displayName},
 
 Your account creation is under review.
 Could you tell us the motivation behind your registration?
 
 Kind Regards
 {AdminName}
 
-' => 'Здравствуйте {displayName}, ваша учётная запись находится на рассмотрении. Не могли бы вы рассказать нам о причинах вашей регистрации? С уважением, {AdminName}',
+' => 'Здравствуйте {displayName}!
+Ваша учётная запись находится на рассмотрении. Не могли бы Вы рассказать нам о причинах Вашей регистрации?
```

Comment:
> I think we should keep newlines as in the original text

@yurabakhtin 
For keep same amount of lines in file?

Reply:
> To keep same amount of lines in file?

@hide-me No, it doesn't matter what amount of lines will be in the translation files.
I just see the original text has new lines, so it would be better to keep the same new lines.
Such texts are used for some email messages, and I think text with new line more readable.

Reply:
Good, sure! Thank you for the answer.

> No, it doesn't matter what amount of lines will be in the translation files.

Yes, I know, but maybe this file would be a bit prettier when someone decide to compare lang files string by string or something like this 😄